packages/usa-accordion/src/styles/_usa-accordion.scss

```diff
@@ -1,9 +1,32 @@
 @use "uswds-core" as *;
 
 // Variables
+$accordion-context: "Accordion";
+
 $accordion-border: units($theme-accordion-border-width) solid
   color($theme-accordion-border-color);
 
+$accordion-background-active-color: next-token(
+  $theme-accordion-button-background-color,
+  "darker"
+);
+
+$accordion-button-darkmode: is-input-darkmode(
+  $theme-accordion-button-background-color
+);
+
+$accordion-button-unopen-icon: if(
+  $accordion-button-darkmode,
+  "usa-icons-bg/add--white",
+  "usa-icons/add"
+);
```

Comment:
Might be an edge case, but `yellow-20v` fails color contrast standards.

I've tried recreating with similarly toned colors and haven't found any others that fail color contrast

![Screenshot 2023-05-04 at 10 34 08 AM](https://user-images.githubusercontent.com/25211650/236248455-edb92c31-b70f-4992-89ea-e218fbb797b5.png)

Reply:
@amyleadem thanks for the suggestion, I'm using `calculate-grade()` now instead of checking for lightness [9dd8217].